### PR TITLE
Add prow job to publishing csi-proxy binary

### PR DESCRIPTION
**Pull request by jingxu97**
Add a prow job to publish csi-proxy binary

csi-proxy PR to add cloudbuild.yaml https://github.com/kubernetes-csi/csi-proxy/pull/53

### CODE REVIEW

**Comment by jingxu97** [comment]
cc @verult

File: config/jobs/image-pushing/k8s-csi-proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+postsubmits:
```
Comment (msau42) [inline config/jobs/image-pushing/k8s-csi-proxy.yaml:1]: @pohly has a PR to autogenerate this for all the csi repos in https://github.com/kubernetes/test-infra/pull/17741.

I've asked him to cover csi-proxy in his PR.

Comment (jingxu97) [reply config/jobs/image-pushing/k8s-csi-proxy.yaml:1]: got it. Thanks!

**Comment by spiffxp** [comment]
/close
Looks like this job has been generated
https://github.com/kubernetes/test-infra/blob/ba4a2b49849bccc2790f33b30d48a415d0ca1499/config/jobs/image-pushing/k8s-staging-csi.yaml#L120-L148

**Pull request closed without merging**